### DISCUSSION TITLE
平常寫文件會用Word、Notion、HackMD或是Medium?

### DISCUSSION THREAD

**Discussion by Tchang323** [1101-GS4719-C/Python-Community-GS4719-C#2208] [General]
我目前使用過多種線上文件編輯工具，但各有優缺點。以下是我的體驗與心得：

Notion：我平常使用最頻繁的工具，因為操作界面易懂，排版美觀，且可以自動調整內容大小轉換成PDF，功能十分豐富。然而，它的缺點在於圖片無法直接複製，必須先下載才能使用。此外，共享的連結通常要求對方也必須登入才能查看，有點不方便。

HackMD 和 Medium：這兩者更適合資訊分享，尤其在程式碼展示和協作方面表現出色，但相對於Notion，它們的排版和文件編輯靈活度略顯不足。

Word：作為傳統的文字處理工具，Word的優勢在於它的正式性，許多正式文件仍然依賴Word編寫。然而，Word的功能偏向純文字處理，缺乏更多現代化的協作與多媒體支持。

好奇大家平常較常用哪個?或是可以分享自己喜歡用的工具XD

**Comment by 112121py** [comment]
我在寫一些作業報告或是比較正事需要繳交的東西，我會使用word來處理，畢竟要繳交或是列印大家還是比較接受word。在作筆記或是處理專題的東西我會使用hackmd比較多，因為真的蠻方便做一些簡單的排版，還有放程式碼等等，需要共編或分享也蠻方便的，只是手機或平板要編輯會比較難用qq

**Comment by poyen0806** [comment]
我在寫作業的時候會用word，感覺比較正式，而且還是我大微軟帝國的產物，但如果是做課堂筆記，或是紀錄平常查到的資料，我會用hackmd，畢竟markdown真的很香，有人問我為什麼不用notion，明明notion也能用markdown，我的回答也很簡單，啊我就不會用啊。不過在處理團隊事務的話，我還是會選擇notion，比較好分配工作，啊當然，有gitlab的話我會優先gitlab。

**Comment by ShuliANN** [comment]
平常寫報告時我會用Word，自己在記事情和規劃行程時會用Notion，而在閱覽一些別人的文章時喜歡使用Medium，上面的知識和經驗分享都很受用！！！

**Comment by Denny3908** [comment]
平常交作業會用 Word 和 Latex，比較方便做正式的排版和寫 reference。
自己作筆記都用 Notion，可以用 markdown、寫數學式和程式碼都很方便，還可以自己寫一些功能。
團隊協作很大部分是用 google 體系的東西，功能很爛但重要的是大家會用啊XD

**Comment by christaaah** [comment]
平時寫作業會用word來寫，因為可以處理一些正式的排版，但是平時做筆記的時候都是用HackMD寫筆記，因為放圖片、程式碼都可以很方便，而且不用擔心排版問題，很輕鬆就能讓筆記看起來很工整，有時候要和同學一起協作的時候也很方便、或是做一些教材也蠻方便的。

**Comment by forestdew507** [comment]
團隊合作 : Notion > 就算對方沒有帳號，選擇 publish 候有連結的人都可以有檢視功能，基本上討論紀錄等基本資訊就很好同步了 ! + Google Drive > 大家都熟悉的共用/編軟體
閱讀深度文章 : Medium > 不得不說 Medium 讓人一再見識知識海的廣大XD
交作業 : Word > 排版不會亂掉地讓人安心

FYI 看了這篇討論和留言受益不少哈哈，沒想到那麼多人使用 HackMD 作筆記，我過去都只有在 HackMD 上瀏覽別人的筆記而已~

**Comment by Jinyuu17** [comment]
要很完美的排版，我會使用 Overleaf 撰寫 LaTeX 來生成 PDF 檔案。但缺點是要打一堆程式碼，若文件的內容是中文，就要在中英文間做切換，相當的麻煩。

**Comment by 113607514** [comment]
我平常寫文件時會使用多種工具，如 Word、Notion、HackMD 和 Medium。Word 是我最常用的文字處理工具，方便進行格式編輯；Notion 則是我用來整理筆記和計畫的好幫手，能夠把資料以不同方式呈現；HackMD 讓我輕鬆地進行協作寫作，適合與他人共同編輯；而 Medium 則是一個很好的平台，讓我能分享我的想法和故事，觸及更廣泛的讀者群。這些工具各有特點，能滿足我不同的寫作需求。

**Comment by yuan3x2** [comment]
我在個人作業中通常會使用 Word，比較正式，而在分組作業中則偏好使用 HackMD，因為它提供了便捷的協作編輯功能，並且相比 Word，其格式更為簡潔，適合快速編排和內容整理。
